scc/cert/member_json.go

```diff
@@ -0,0 +1,48 @@
+package cert
+
+import (
+	"fmt"
+
+	"github.com/0xsoniclabs/sonic/scc"
+	"github.com/0xsoniclabs/sonic/scc/bls"
+	"github.com/0xsoniclabs/sonic/utils/jsonhex"
+)
+
+// MemberJson is a JSON friendly representation of a scc.Member.
+type MemberJson struct {
+	PublicKey         jsonhex.Bytes48 `json:"publicKey" gencodec:"required"`
+	ProofOfPossession jsonhex.Bytes96 `json:"proofOfPossession" gencodec:"required"`
+	Weight            uint64          `json:"weight" gencodec:"required"`
+}
+
+// String returns the JSON string representation of the MemberJson.
+func (m MemberJson) String() string {
+	return fmt.Sprintf(`{"publicKey":"%v","proofOfPossession":"%v","weight":%d}`,
+		m.PublicKey, m.ProofOfPossession, m.Weight)
```

Comment:
nit: this is likely equivalent to what I would get if I print the object using "%v" or "%+v" without a `String()` implementation.

Reply:
almost. using "%+v" does not add the needed `"` for all the names and non numerical values, which we need. Also does not add a `,` after each value.